### PR TITLE
docs: Update ko-kr README.md

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**
* removed not viewable video link (f1d30750d7ae417eddbdcba9f60a18704125d3f0)
* fixed wrong directory path (877a58fb5d2c96362c520d39f6516d02b8ace4e7)
* introduce oopsy click-to-copy function (a5dcec0a89f95422a8154c398c0066e0d85d2a84)
  * Chinese translation needed
  * I'm not English native, anyone can suggest better English description.
* update Jobs:drg(Korean) description for EW (6a9c42273cc3dd4ac3a874665bac61126fc869b9)
  * Only Korean dragoon description is updated. (I only know Korean dragoon.)
  * Other jobs needed to be updated.
  * Other languages are still informing ShB version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,6 +338,8 @@ Mistake triggers are specified for individual fights in the [ui/oopsyraidsy/data
 
 ![oopsy screenshot](screenshots/promo_oopsy.png)
 
+You can copy oopsy logs to clipboard by clicking it. (You have to uncheck `Enable Clickthrough` checkbox from the overlay option.)
```
Comment (quisquous) [inline README.md:341]:
```suggestion
You can copy oopsy lines to the clipboard by clicking them.
(You may need to uncheck `Enable Clickthrough` checkbox from the OverlayPlugin option.)
```

Some small edits, and also adding a line break to follow https://sembr.org/.

**Comment by quisquous** [comment]
Thanks for all these fixes and improvements!

**Comment by Bing-su** [comment]
이런게 있었다니!

File: README.md (hunk 1 above)
Comment (Jaehyuk-Lee) [reply README.md]: Oh... I didn't know sembr. I'll review again with that.

**Comment by Jaehyuk-Lee** [comment]
I couldn't apply sembr strictly. 😅 But tried.

**Comment by quisquous** [comment]
Thanks! I think it doesn't have to be done strictly, but I think just better (where possible) to break up long lines (at least at sentences) to make diffs easier to read.

**Pull request merged**